docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -479,10 +434,41 @@ class Sidebar extends React.Component<any, any> {
           />
           <Text color="white" content={pkg.version} size="medium" styles={logoStyles} />
         </Segment>
+        <Flex column>
+          <a
+            href={constants.repoURL}
+            target="_blank"
+            rel="noopener noreferrer"
+            style={topItemTheme}
+          >
+            <Box>
+              GitHub
+              <Icon name="github" styles={{ float: 'right' }} />
+            </Box>
+          </a>
+          <a href={changeLogUrl} target="_blank" rel="noopener noreferrer" style={topItemTheme}>
+            <Box>
+              CHANGELOG
+              <Icon name="file alternate outline" styles={{ float: 'right' }} />
+            </Box>
+          </a>
+          <Input
+            style={topItemTheme}
```

Comment:
Please always use `styles` instead of `style` with Stardust components. Also, instead of padding, try applying margin and everything should work. The icon inside the menu is positioned absolutelly, that's why padding is not working: https://codesandbox.io/s/stardust-ui-example-i4e6k

Reply:
I think that in this case, the code will look cleaner if I stick to padding.